MIGRATION.md

```diff
@@ -374,6 +375,12 @@ The `IconButton` doesn't have any deprecated props but it now uses the new `Butt
 
 In Storybook 8.0 we are introducing a new icon library available with `@storybook/icons`. We are deprecating the `Icons` component in `@storybook/components` and recommend that addon creators and Storybook maintainers use the new `@storybook/icons` component instead.
 
+#### StoryStore methods deprecated
+
+The following methods on the `StoryStore` are deprecated and will be removed in 9.0:
+  - `store.fromId()` - please use `store.loadStory()` instead.
+  - `store.raw()` - please use `store.extract()` instead.
```

Comment:
I'm not sure how common it is that addons use these. We could consider pushing this into 7.6.0 and removing in 8.0, but there's no a lot of cost in keeping them, and @ndelangen and I decided to play it safe.

Reply:
Apart from React Native e.g. on-device-controls using `fromId`, but I guess this won't affect them until Preview is integrated, there is the Zeplin addon:
https://github.com/zeplin/storybook-inspector/blob/6e5f8b757803d700ba828939ada156d5a316cec4/src/story.ts#L85